README.md

```diff
@@ -26,29 +26,31 @@
 </p>
 
 ## **Table of contents**
-1. [**What is Horusec?**](#What-is-Horusec?)
-2. [**Getting started**](#Getting-started)
-    1. [**Requirements**](#Requirements)
-    2. [**Installing Horusec**](#Installing-Horusec)
-    3. [**Usage**](#Usage)
-        1. [**CLI Usage**](#CLI-Usage)
-        2. [**Using Docker**](#Using-Docker)
-        3. [**Older versions**](#Older-versions)
-        4. [**Using Horusec-Web application**](#Using-Horusec-Web-application)
-        5. [**Using Visual Studio Code**](#Using-Visual-Studio-Code)
-        6. [**Using the Pipeline**](#Using-the-Pipeline)
-4. [**Contributing**](#Contributing)
-5. [**Roadmap**](#Community)
-6. [**License**](#License)
-7. [**Community**](#Community)
+### 1. [**About**](#about)
+### 2. [**Getting started**](#getting-started)
+>#### i.   [**Requirements**](#requirements)
+>#### ii.  [**Installation**](#installing-horusec)
+>#### iii. [**Usage**](#usage)
+>##### i. [**CLI Usage**](#cli-usage-1)
+>##### ii. [**Using Docker**](#using-docker-1)
+>##### iii. [**Older versions**](#older-versions-1)
+>##### iv. [**Using Horusec-Web application**](#using-horusec-web-application-1)
+>##### v.  [**Using Visual Studio Code**](#using-visual-studio-code-1)
+>##### vi. [**Using the Pipeline**](#Using-the-Pipeline)
```

Comment:
```suggestion
> #### 2.1. [**Requirements**](#requirements)
> #### 2.2. [**Installation**](#installing-horusec)
> #### 2.3. [**Usage**](#usage)
>> ##### 2.3.1. [**CLI Usage**](#cli-usage-1)
>> ##### 2.3.2. [**Using Docker**](#using-docker-1)
>> ##### 2.3.3. [**Older versions**](#older-versions-1)
>> ##### 2.3.4. [**Using Horusec-Web application**](#using-horusec-web-application-1)
>> ##### 2.3.5. [**Using Visual Studio Code**](#using-visual-studio-code-1)
>> ##### 2.3.6. [**Using the Pipeline**](#Using-the-Pipeline)
```

Roman numbers can get confusing here wouldn't it be better to stick to the standard numbering?

Reply:
I prefer it as well, however it was decided Romans, but we can change! ;)